demos/tests/args.py

```diff
@@ -106,3 +115,11 @@ def __init__(self, file_name):
 
     def resolve(self, context):
         return str(context.source_dir / self.file_name)
+
+
+class OmzDirArg:
+    def __init__(self, file_name):
+        self.file_name = file_name
+
+    def resolve(self, context):
+        return str(context.omz_dir / self.file_name)
```

Comment:
I don't think this class needs to exist. The OMZ directory is effectively a constant; you could just define `OMZ_DIR = Path(__file__).resolve().parents[2]`, and use `OMZ_DIR / 'foo/bar'` as the argument.